changelog.md

```diff
@@ -1,3 +1,22 @@
+## [0.34.1]
+
+### Breaking Changes
+* None.
+
+### New Features
+* None.
+
+### Enhancements
+* Updated version of gRPC dependencies to `1.59.1`.
+* Updated version of GraphQL-Kotlin dependencies to `6.5.6`. From version `7.0.0` onwards, [GraphQL-Kotlin is only
+  supported on Java 17+](https://github.com/ExpediaGroup/graphql-kotlin/releases/tag/7.0.0).
```

Comment:
critical: An update that requires us moving to a new JVM version is not an enhancement, it is a breaking change (and a major one for that matter).

Not only is this a breaking change, it will also need to be confirmed that we can get our customers to actually install a compatible version, or at the very least give them a reasonably large lead time on when we will be requiring it.

@kgreav This is probably something that needs to be discussed

Reply:
@Lax125 Nothing to do here for now, we just need to be warry of this when we do decide to move.